### PR TITLE
refactor: new alert components

### PR DESCRIPTION
**Pull request by toyota-corolla0**
addresses #30

### CODE REVIEW

File: src/components/alerts.rs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+use leptos::*;
+
+#[component]
+pub fn NoteBlock(#[prop(into)] text: String) -> impl IntoView {
+    view! {
+        <div class="bg-yellow-100 border-l-4 border-yellow-500 text-yellow-700 p-4 mb-8 w-full" role="alert">
+          <p class="font-bold">Note</p>
+          <p>{ text }</p>
```
Comment (elsirion) [inline src/components/alerts.rs:8]: Happy to merge like this, but wondering if the content should just be arbitrary child HTML elements? Does the current form support links, bold text etc.?

**Review by elsirion — APPROVED** [review]

**Comment by elsirion** [comment]
Thx for working on webimint!

File: src/components/alerts.rs (hunk 1 above)
Comment (toyota-corolla0) [reply src/components/alerts.rs:8]: Great point! Should line 8 include <p></p>?

Comment (elsirion) [reply src/components/alerts.rs:8]: I think so, by default we only want to display text but want to keep the option to adding styling, links, …

**Comment by elsirion** [comment]
There seems to be a `children` property for what we want to do https://docs.rs/leptos/latest/leptos/attr.component.html

**Review by elsirion — APPROVED** [review]

**Pull request merged**